components/AppLayout/AppLayout.js

```diff
@@ -19,6 +19,9 @@ const USER_QUERY = gql`
       id
       name
       avatarUrl
+      avatarType
```

Comment:
This actually comes from `Avatar` inside `AppHeader` and `AppSidebar`.

Since we follow [fragment colocation](https://www.apollographql.com/docs/react/data/fragments/#creating-colocated-fragments) in this repo, we may need to

- Declare static member `.fragments.User` for `AppSidebar` and `AppHeader` that includes `Avatar.fragments.AvatarData`, since these components are using `<Avatar>`
- AppLayout should use `AppSidebar.fragments.User` here. (apollo-client will perform dedup on the fields)

In this way the dependency of GraphQL fields can be tracked on component basis.